app/src/js/services/Droplet.js

```diff
@@ -1,17 +1,71 @@
 angular.module('media_manager')
 .service('Droplet', ['$timeout', '$log', '$q', 'AppConfig', function($timeout, $log, $q, AppConfig){
   var ds = this;
+  var ONE_MEGABYTE = 1000000; // bytes
```

Comment:
I know this is not the same thing, and this has a use... but it still reminds me of: https://www.reddit.com/r/programminghorror/comments/4eaanc/what_if_one_day_5_6_better_safe_than_sorry/